src/poc/miner_onion_server.erl

```diff
@@ -391,3 +392,15 @@ try_decrypt(IV, OnionCompactKey, OnionKeyHash, Tag, CipherText, ECDHFun, Chain)
             {error, too_many_pocs}
     end.
 
+-spec spreading(integer()) -> string().
+spreading(Len) when Len < 12 ->
```

Comment:
Just to confirm, on equality we want to go to the next matching fun?

Reply:
Right. For example, if the packet size is 12 bytes, we should be using SF9BW125